conf/locale

```diff
@@ -0,0 +1 @@
+../freetextresponse/locale/
```

Comment:
Eventually we found a reason for this.  Translations were not found without adding xblock plath to LOCALE_PATHS if the symlink was not in place.  Symlink allows us to only add the xblock to OPTIONAL_APPS